menu.xml

```diff
@@ -138,4 +138,6 @@
 			<Reset_to_default_settings text="Cancel"></Reset_to_default_settings>		
 		</Other_Settings>
 	</menu>
+	<menu text="Shutdown">
```

Comment:
I think this option should be moved deeper in case you need to add, for example, a restart option. Also, confirmation is a must have.